src/Pim/Component/Catalog/Normalizer/Indexing/Product/PropertiesNormalizer.php

```diff
@@ -77,4 +97,29 @@ public function supportsNormalization($data, $format = null)
     {
         return $data instanceof ProductInterface && 'indexing' === $format;
     }
+
+    /**
+     * @param ProductInterface $product
+     *
+     * @return array
+     */
+    protected function getAssociatedProducts(ProductInterface $product)
+    {
+        $isAssociated = [];
+
+        $associations = $this->associationRepository->getAssociationsContainingProduct($product);
```

Comment:
No, it will indeed give me the associations, but not the ones I want.
What I want are the associations the product is IN, not the ones it contains.